selectrum.el

```diff
@@ -778,6 +779,15 @@ into the user input area to start with."
           (substitute-command-keys
            "[\\[selectrum-select-additional] enabled] ")
           (text-properties-at (point)))))))
+  (when crm-completion-table
+    (let ((inhibit-read-only t))
+      (save-excursion
+        (minibuffer-prompt-end)
+        (when (search-backward ":" nil t)
+          (insert
+           (apply #'propertize
+                  " [one or more]"
+                  (text-properties-at (point))))))))
```

Comment:
This is a better way of indicating multiple selection. You can get rid of my previous attempt based on `substitute-command-keys`.

Reply:
Done, also replaced the old multi selection feature with the current one entirely